specs/006-curation-pipeline/spec.md

```
@@ -0,0 +1,372 @@
+```
+
+**Invariants**:
+- `endpoint` must be unique per user (upsert on re-registration).
+
+---
+
+## 2. State Machines
+
+### 2.1 Proposal State Machine
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)